src/dune_rules/stanzas/copy_files.ml

```diff
@@ -22,8 +23,16 @@ let long_form =
   and+ mode = field "mode" ~default:Rule.Mode.Standard (check >>> Rule_mode_decoder.decode)
   and+ enabled_if = Enabled_if.decode ~allowed_vars:Any ~since:(Some (2, 8)) ()
   and+ files = field "files" (check >>> String_with_vars.decode)
+  and+ only_sources = field_b "only_sources"
```

Comment:
This field should be available starting 3.14

Reply:
https://github.com/ocaml/dune/pull/9827/commits/12cd755e85b21597f648727f66f7edb1aec458f2

Reply:
n/m this was a larger change that I expected, created https://github.com/ocaml/dune/pull/9840 for it, should be merged first.